advocacy_docs/supported-open-source/postgresql/security_guidance/advanced.mdx

```diff
@@ -0,0 +1,190 @@
+---
+title: "Advanced"
```

Comment:
```suggestion
title: "PostgreSQL Security 301: Advanced Security Guide"
```

The title should be the title so it gets H1 heading (for SEO) - and delete the followin H2 title, and maybe uprate the H3 headings to H2